purchasables.md

```diff
@@ -1,14 +1,132 @@
+  "isDefault": "1",
+  "sku": "ANT-001",
+  "price": 20,
+  "sortOrder": "1",
+  "width": null,
+  "height": null,
+  "length": null,
+  "weight": null,
+  "stock": "0",
+  "hasUnlimitedStock": "1",
+  "minQty": null,
+  "maxQty": null,
+  "deletedWithProduct": false,
+  "id": "30",
+  "tempId": null,
+  "draftId": null,
+  "revisionId": null,
+  "uid": "076305ef-17c5-4433-99de-bf5fb090f704",
+  "fieldLayoutId": null,
+  "contentId": "23",
+  "enabled": "1",
+  "archived": "0",
+  "siteId": "1",
+  "title": "A New Toga",
+  "slug": "ant-001",
+  "uri": null,
+  "dateCreated": "2020-06-29T09:12:02-07:00",
+  "dateUpdated": "2020-06-29T09:12:03-07:00",
+  "dateDeleted": null,
+  "trashed": false,
+  "propagateAll": false,
+  "newSiteIds": [],
+  "resaving": false,
+  "duplicateOf": null,
+  "previewing": false,
+  "hardDelete": false,
+  "ref": null,
+  "status": "enabled",
+  "structureId": null,
+  "url": "https://foo.dev/shop/products/ant-001?variant=30",
+  "isAvailable": true,
+  "isPromotable": true,
+  "shippingCategoryId": 1,
+  "taxCategoryId": 1,
+  "onSale": false,
+  "cpEditUrl": "#",
+  "product": {
+    "postDate": "2020-06-29T09:12:02-07:00",
+    "expiryDate": null,
+    "typeId": "1",
+    "taxCategoryId": "1",
+    "shippingCategoryId": "1",
+    "promotable": "1",
+    "freeShipping": null,
+    "enabled": "1",
+    "availableForPurchase": true,
+    "defaultVariantId": "30",
+    "defaultSku": "ANT-001",
+    "defaultPrice": "20.0000",
+    "defaultHeight": "0.0000",
+    "defaultLength": "0.0000",
+    "defaultWidth": "0.0000",
+    "defaultWeight": "0.0000",
+    "taxCategory": null,
+    "name": null,
+    "id": "29",
+    "tempId": null,
+    "draftId": null,
+    "revisionId": null,
+    "uid": "71a2efe8-163f-4014-9ffc-b8272d469a7e",
+    "fieldLayoutId": null,
+    "contentId": "22",
+    "archived": "0",
+    "siteId": "1",
+    "title": "A New Toga",
+    "slug": "ant-001",
+    "uri": "shop/products/ant-001",
+    "dateCreated": "2020-06-29T09:12:02-07:00",
+    "dateUpdated": "2020-06-29T09:12:02-07:00",
+    "dateDeleted": null,
+    "trashed": false,
+    "propagateAll": false,
+    "newSiteIds": [],
+    "resaving": false,
+    "duplicateOf": null,
+    "previewing": false,
+    "hardDelete": false,
+    "ref": null,
+    "status": "live",
+    "structureId": null,
+    "url": "https://foo.dev/shop/products/ant-001"
+  },
+  "description": "A New Toga",
+  "purchasableId": "30",
+  "options": [],
+  "sales": []
+}
+```
 
-A purchasable:
+:::
 
-- is an [element type](https://docs.craftcms.com/v3/extend/element-types.html)
-- implements [`craft\commerce\base\PurchasableInterface`](api:craft\commerce\base\PurchasableInterface)
-- should extend [`craft\commerce\base\Purchasable`](api:craft\commerce\base\Purchasable)
+This can be particularly helpful, for example, if `getPurchasable()` returns `null` on the line item and you need to display information from the purchasable in a customer’s order history.
```

Comment:
Mention that the purchasable might be null because it was deleted since the order was made. This will only happen for completed orders, since the line item would be removed from the cart if it no longer existed (was deleted) while the user was in the checkout process.

Reply:
👍